modules/apps/login/login-web/src/main/java/com/liferay/login/web/internal/portlet/action/LoginMVCActionCommand.java

```diff
@@ -121,6 +133,41 @@ protected void doProcessAction(
 					SessionErrors.add(actionRequest, exception.getClass());
 				}
 			}
+			else if (exception instanceof UserLockoutException.PasswordPolicyLockout) {
+				User user = _getUser(actionRequest);
+
+				Company company = themeDisplay.getCompany();
+
+				PortletPreferences portletPreferences = actionRequest.getPreferences();
+
+				String languageId = _language.getLanguageId(actionRequest);
+
+				String emailFromName = portletPreferences.getValue(
+					"emailFromName", null);
+				String emailFromAddress = portletPreferences.getValue(
+					"emailFromAddress", null);
+				String emailToAddress = user.getEmailAddress();
+
+				String emailParam = "emailPasswordSent";
+
+				if (company.isSendPasswordResetLink()) {
+					emailParam = "emailPasswordReset";
+				}
+
+				String subject = portletPreferences.getValue(
+					emailParam + "Subject_" + languageId, null);
+				String body = portletPreferences.getValue(
+					emailParam + "Body_" + languageId, null);
```

Comment:
We need to send out the lockout notification email, we should load those templates instead from portal.properties.